mvp-spec/operator-api.md

```diff
@@ -16,13 +16,13 @@ These endpoints need to support both the "3PC" and "no 3PC" contexts.
 
 In practice, this will translate into endpoints available under different root paths. Example paths are specified in the last column of the table.
 
-| Endpoint                                | Input*                  | Output                                                | Description                                                                                                                                                                               | Notes                                                                                                                                                                                                                                                                                   | Rest                        | Redirect                        |
```

Comment:
Here, "REST" is inaccurate. In REST, GET is safe (i.e read-only) whereas we potentially create a resource in our case. 

Possible change:
1. *Direct*. 
   * Pro: Symmetry with "Redirect". 
   * Cons: Subject to typo/error because the two words are closed.
2. *On Site*
   * Pro: Explicite on how it is used
   * Cons: May not match all the case.
3. *Direct Data*
   * Pro: agnostic of the format, symmetry

What do you think ? :-)

Reply:
I think that you have improved your API with your last commit and It is a good thing. It is more idiomatic now. 

If I check this [maturity model](https://en.wikipedia.org/wiki/Richardson_Maturity_Model), I guess that we are between level 1 and level 2 of REST. So it may not be RESTful. It is not really clear to me if our API is resource-oriented. The `GET /v1/id-prefs` isn't idempotent. However, do we really care? We don't need to be RESTful. REST doesn't exist for itself. It is a set of principles to ease scalability, uniformity, and layered architectures. You can have those properties without being RESTful.

So regarding this, I am still tempted to remove the REST word from the header of the table.

Reply:
As you say, REST doesn't even really exist so I bet most of the APIs in the wild that call themselves "REST" aren't RESTfull 😁
In that sense, I don't think it's a big deal to leave the REST keyword in this table. Specially, I'm concerned renaming it "direct" or "on site" would bring more confusion than clarity, don't you think?
Unless we find a name that we agree on, I would leave REST for now

Reply:
After our last discussion, I will leave REST in this table, but add a quick "disclaimer".